composer.json

```diff
@@ -36,7 +36,7 @@
     },
     "extra": {
         "branch-alias": {
-            "4.x-dev": "4.2.x-dev"
+            "4.x-dev": "4.4.x-dev"
```

Comment:
Not sure why we still have branch aliases here. I thought we got rid of all of those things.

Reply:
seems like we got rid of the a.b.x-dev branch-aliases, on a bunch of modules we still have the "dev-master": "5.x-dev", idea being that if (for whatever reason) you try to install the next major version you get the master branch.  I'm not entirely sure why we even bother with this, though I've gotten feedback from other core committers that they're sometimes useful

seems like this should be changed to "dev-master": "5.x-dev" though